README.rst

```diff
@@ -472,7 +472,7 @@ template ``context_processors`` in ``settings.py`` as shown below.
 ``register_menu_group``
 ^^^^^^^^^^^^^^^^^^^^^^^
 
-It allows us to register different menu item or group at different position in the Main Navigation Menu.
+It allows to register different menu item or group at the specified position in the Main Navigation Menu.
```

Comment:
"to register different menu item", wouldn't it be better "to register a new menu item"?